src/main/java/lotto/LottoStatistics.java

```diff
@@ -6,8 +6,8 @@
 import static lotto.Lotto.LOTTO_PRICE;
 
 public class LottoStatistics {
-    public static int getLottoBuyingCount(int money) {
-        return Math.round(money / LOTTO_PRICE);
+    public static int getLottoBuyingCount(Money money) {
+        return Math.round(money.value() / LOTTO_PRICE);
```

Comment:
```suggestion
        return money.value() / LOTTO_PRICE;
```

Math.round 메서드가 필요한가요?
그냥 나눠도 되지 않나요?